Dockerfile

```
@@ -1,7 +1,8 @@
-FROM node:24-slim AS base
+FROM node:25-slim AS base
 ENV PNPM_HOME="/pnpm"
 ENV PATH="$PNPM_HOME:$PATH"
-RUN corepack enable
+RUN apt update && apt install --yes wget && rm -rf /var/lib/apt/lists/*
```

Hadolint (2.14.0)

[warning] Do not use apt as it is meant to be an end-user tool, use apt-get or apt-cache instead

(DL3027)